core/field_multilineinput.js

```diff
@@ -286,9 +359,14 @@ Blockly.Css.register([
   '.blocklyHtmlTextAreaInput {',
     'font-family: monospace;',
     'resize: none;',
-    'overflow: hidden;',
+    'overflow: scroll;',
+    '-ms-overflow-style: none;',
```

Comment:
The default should show the scrollbars since that is the typical paradigm for a scrollable text area. You can remove them for your project in your own css if you really don't want to show them.

Reply:
I will fix this too in a few days.   
 
I cant just enable them because it will result in improper block size, and text will break down in places where it should not.      
Last time I tried updating block size in a way to include scrollbar width by using `clientWidth`, got some strange glitches, so I need a little more time to figure this one out.